lib/diagnostic.rb

```diff
@@ -43,6 +43,10 @@ def transform(words)
   # replacing the trailing newline character with a
   # space and return the result.
   def read_file(filename)
-    # Your code goes here
+    File.open('filename') do |file|
+  file.each_line do |line|
+    puts line
+   end
```

Comment:
You opened the file correctly, but you didn't do the correct behavior with each line.

> "Concatenate the lines together after replacing the trailing newline character with a		   space and return the result."